src/main/java/org/apache/commons/lang3/CharUtils.java

```diff
@@ -305,9 +306,10 @@ public static int toIntValue(final Character ch, final int defaultValue) {
      * @param ch  the character to convert
      * @return a String containing the one specified character
      */
+    @SuppressWarnings({"index:array.access.unsafe.low"}) // #1: CHAR_STRING_ARRAY has length 128, and ch < 128, also ASCII code is not negative
     public static String toString(final char ch) {
```

Comment:
Shouldn't this be `@NonNegative char ch` then? And then there would be no need to suppress the warning.